### ISSUE TITLE
Relationships Cache is not invalidated when a bundle with related content is uploaded

### ISSUE THREAD

**Issue by nollymar** [dotCMS/core#17528]
After uploading a bundle that contains several pieces of contents with relationships, the related content list of any of the content might get dirty and outdated. As a workaround, the cache has to be flushed to get the real list.

I can provide a bundle to reproduce the issue

**Comment by nollymar** [comment]
Fixed. PRs: https://github.com/dotCMS/core/pull/17538 https://github.com/dotCMS/enterprise-2.x/pull/689

**Comment by dsilvam** [comment]
Note to QA: Expected Results after importing bundle
```
Movie 1:
Sequel: Movie 2
Rating:A

Movie 2:
Sequel: Movie 3
Prequel: Movie 1
Rating: B

Movie 3:
Prequel: Movie 1 y 2
Rating: A
```

**Comment by dsilvam** [comment]
Passed Internal QA Release: all related content showing correctly when compared to expected list above.

**Comment by bryanboza** [comment]
@nollymar can you upload the bundle here? This to make the same test with master after the release. Thanks

**Comment by nollymar** [comment]
This is the bundle I used [test-95230db4-779b-475f-baf4-94460714c59e.tar.gz](https://github.com/dotCMS/core/files/3789303/test-95230db4-779b-475f-baf4-94460714c59e.tar.gz)

**Comment by bryanboza** [comment]
Fixed, tested on release-5.2.1 // Postgres // FF

**Comment by dsilvam** [comment]
Passed Internal QA: working as expected in master
